R/read_asc_PerkinElmer.R

```diff
@@ -60,3 +60,48 @@ read_asc_PerkinElmer <- function(file = stop("filename or connection needed"),
 # Unit tests -----------------------------------------------------------------
 
 # FIXME: add unit tests, if possible.
+hySpc.testthat::test(read_asc_PerkinElmer) <- function() {
+  context("read_asc_PerkinElmer")
+
+  path <- system.file("extdata/fileio/txt.PerkinElmer",package = "hySpc.read.txt")
+  flu_file <- paste0(path, c("/flu1.txt", "/flu2.txt",
+                             "/flu3.txt", "/flu4.txt", "/flu5.txt", "/flu6.txt"))
+
+  test_that("PerkinElmer .txt labels are correct", {
+    for(flu in flu_file){
```

Comment:
Add space between `for` and `(`

Reply:
And space before `{`

Reply:
I recommend applying  `styler` in the file or selection after you write the code so that it corrected the code style automatically.